shared/src/akri/configuration.rs

```diff
@@ -45,6 +46,13 @@ fn default_action() -> FilterType {
     FilterType::Include
 }
 
+/// This defines the Zeroconf data stored in the Configuration
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct ZeroconfDiscoveryHandlerConfig {
+    pub filter: String,
```

Comment:
it looks like the zeroconf crate supports several filters and this is leveraging that ... is the crate's filter syntax well known?  if not, we might consider using the filter constructs that onvif and udev use (which allow inclusion and exclusion).

Reply:
Oh, I like this. I wish I'd thought of it. It's cleaner and clearer than what I have. I'll incorporate. Thank you.

Reply:
I think much better, thank you!

Fixed: https://github.com/deislabs/akri/pull/163/commits/ea19332ca2ba670dc79904d646929b8d5f4f82e7